services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/domain/Hanke.kt

```diff
@@ -68,6 +66,12 @@ data class Hanke(
         return alueet.map { it.tarinaHaitta }.filterNotNull().toSet()
     }
 
+    val alkuPvm: ZonedDateTime?
```

Comment:
Should these have one of the `@JsonView(ChangeLogView::class)` or `@JsonView(NotInChangeLogView::class)` annotations?